openreview/journal/invitation.py

```diff
@@ -3812,7 +3813,21 @@ def set_review_invitation(self):
 
         if self.journal.get_review_additional_fields():
             for key, value in self.journal.get_review_additional_fields().items():
-                invitation['edit']['note']['content'][key] = value if value else { "delete": True }         
+                invitation['edit']['note']['content'][key] = value if value else { "delete": True }
+
+        if self.journal.get_assignment_delay_after_submitted_review() > 0:
+            weeks = datetime.timedelta(weeks=self.journal.get_assignment_delay_after_submitted_review())
+            milliseconds = int(weeks.total_seconds() * 1000)
+            invitation['postprocesses'] = [
```

Comment:
when a postprocesses run?

Reply:
The postprocess will run after a reply is posted to the invitation, so kind of like the process function. However, we can add a delay, so the postprocess will run after the specified delay (in ms). More info here: https://github.com/openreview/openreview-api/pull/630

Reply:
can we always add this post process and if the delay is 0 then we update the pending review edge right away?

then we need to remove the code that updates the edge in the review process